client/src/components/TestSuite/TestSuiteTree/TestGroupTreeItem.tsx

```diff
@@ -30,6 +31,7 @@ const TestGroupTreeItem: FC<TestGroupTreeItemProps> = ({
     <CustomTreeItem
       nodeId={testGroup.id}
       label={<TreeItemLabel runnable={testGroup} />}
+      icon={testGroup.run_as_group ? <CondensedResultIcon result={testGroup.result} /> : null}
```

Comment:
Icon is not appearing in any test suite other than the Standardized Testing API

Reply:
All set, good catch.

![Screen Shot 2022-02-08 at 11 13 29 AM](https://user-images.githubusercontent.com/412901/153028893-ec3e61a4-141d-4e47-b751-178b22931a05.png)